crates/virtio-queue/Cargo.toml

```diff
@@ -0,0 +1,27 @@
+[package]
+name = "virtio-queue"
+version = "0.1.0"
+authors = ["The Chromium OS Authors"]
+description = "virtio queue implementation"
+repository = "https://github.com/rust-vmm/vm-virtio"
+keywords = ["virtio"]
+readme = "README.md"
+license = "Apache-2.0 OR MIT"
+edition = "2018"
+
+[features]
+test-utils = []
+
+[dependencies]
+vm-memory = ">=0.4.0"
+vmm-sys-util = ">=0.4.0"
+log = ">=0.4.6"
+
+[dev-dependencies]
+criterion = "0.3.0"
+vm-memory = { version = ">=0.4.0", features = ["backend-mmap", "backend-atomic"] }
+virtio-queue = { path = ".", features = ["test-utils"] }
```

Comment:
Hmmm, this shouldn't be needed. Why does the crate depend on itself?

Reply:
I was using it to always have the test-utils feature enabled on dev builds, but I think it is not necessary. I'll delete it.